pyuvdata/uvdata/tests/test_uvdata.py

```diff
@@ -5446,23 +5469,34 @@ def test_downsample_in_time_uneven_samples(resample_in_time_file):
         )
     )
 
+    # make sure integration time is correct
+    # in this case, all integration times should be the target one
+    assert np.all(np.isclose(uv_object.integration_time, min_integration_time))
+
     # as usual, the new data should be the average of the input data (3 points now)
     out_wf = uv_object.get_data(0, 1)
     assert np.isclose(np.mean(init_wf[0:3, 0, 0]), out_wf[0, 0, 0])
 
-    return
+    # Compare doing it with n_times_to_avg
+    uv_object2.downsample_in_time(
+        n_times_to_avg=3, blt_order="baseline", minor_order="time", keep_ragged=False
+    )
+    assert uv_object.history != uv_object2.history
+    uv_object2.history = uv_object.history
+    assert uv_object == uv_object2
 
 
 @pytest.mark.filterwarnings("ignore:The xyz array in ENU_from_ECEF")
 @pytest.mark.filterwarnings("ignore:The enu array in ECEF_from_ENU")
-def test_downsample_in_time_uneven_samples_discard_ragged(resample_in_time_file):
+def test_downsample_in_time_uneven_samples_keep_ragged(resample_in_time_file):
```

Comment:
Why did we change from keeping ragged to discarding them? Also, the test's docstring should be updated.

Reply:
For coverage. The test just above this one tests nearly the same thing with `keep_ragged=False`, so I switched this one to `True` to improve the coverage without increasing the number of tests. I'll fix the docstring.